src/SupervisionSuccess/model/produceProjections.js

```diff
@@ -98,6 +98,58 @@ function produceProjections(
     )
   );
 
+  const maxRevocationsByMonth = Array.from({
+    length: months + ADDED_MONTHS,
+  }).map((revocations, month) =>
+    calcRevocations(
+      month,
+      actualImplementationPeriod,
+      revocationA,
+      RAlpha0,
+      revocationsTimescale,
+      50
```

Comment:
We should extract these 50 and -50 numbers out to constants that are shared with the sliders themselves, if possible. That way, if we ever update the sliders to allow, say, +100% and -100%, we only have to update those numbers and not make multiple updates across the codebase.

Reply:
Done